packages/StoryCard/style.js

```diff
@@ -0,0 +1,73 @@
+import styled from 'styled-components'
+import { colorGreyGainsboro, colorTelusPurple, colorWhite } from '@tds/core-colours'
+
```

Comment:
Use REM instead of px when possible.  Some places may not be but font-size, border radius, it makes sense to use REM